augur/generate.py

```diff
@@ -133,15 +150,43 @@ def two_point_template(config):
             print("Missing Nz_type in %s. Quitting." % src)
             raise RuntimeError
         if tcfg["Nz_type"] == "Gaussian":
-            mu, sig = tcfg["Nz_center"], tcfg["Nz_sigma"]
-            zar = np.linspace(max(0, mu - 5 * sig), mu + 5 * sig, 500)
-            Nz = np.exp(-((zar - mu) ** 2) / (2 * sig ** 2))
-            S.add_tracer("NZ", src, zar, Nz)
+            mu, wi = tcfg["Nz_center"], tcfg["Nz_width"]
+            alpha, z0 = tcfg["Nz_alpha"], tcfg["Nz_z0"]
+            sig_z = tcfg["Nz_sigmaz"]
+            zall = np.linspace(0, 4, 1500)
+            mask = (zall > mu-wi/2) & (zall < mu+wi/2)
+            dndz_bin = np.zeros_like(zall)
+            dndz_bin[mask] = srd_dndz(zall[mask], z0, alpha)
```

Comment:
Ok, no, I think if Nz_type is Gaussian, then it should be Gaussian, not SRD.
We need a NZ_type == "SRD2018" or something that will be the SRD assumed. Others should do exactly what is says on the can (i.e. we want to use this also for things not in SRD)

Reply:
Agree with Anze here.